Pipfile

```diff
@@ -10,7 +10,7 @@ name = "test-pypi"
 
 [dev-packages]
 # we fix exact versions as it's sufficient to have at least one set of compatible dependencies for development
-setuptools = "==59.6.0"
+setuptools = "==62.3.2"
```

Comment:
admittedly not related to PR but I needed this upgrade on my system (latest ubuntu)